.github/workflows/test.yml

```
@@ -0,0 +1,174 @@
+        cache: 'npm'
+    
+    - name: Install dependencies
+      run: npm ci
+    
+    - name: Run linter
+      run: npm run lint || true
+      continue-on-error: true
+    
+    - name: Build project
+      run: npm run build
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)